index.md

```diff
@@ -56,20 +56,18 @@ __Technical information__
 
 ## Project and community
 
-Charmed HPC is a project of the [Ubuntu High-Performance Computing
+Charmed HPC is an open-source project of the [Ubuntu High-Performance Computing
 community](https://ubuntu.com/community/governance/teams/hpc).
 Interested in contributing bug fixes, patches, documentation, or feedback?
 Want to join the Ubuntu HPC community? You've come to the right place!
 
 Here's some links to help you get started with joining the community:
 
-<!-- TODO: Add page with contributing guidelines. -->
-<!-- TODO: Add page with more information on how to get support for Charmed HPC. -->
-
-* [Ubuntu Code of Conduct](https://ubuntu.com/community/ethos/code-of-conduct)
-* [Join the conversation on Matrix](https://matrix.to/#/#hpc:ubuntu.com)
+* [Read and follow the Ubuntu Code of Conduct](https://ubuntu.com/community/ethos/code-of-conduct)
+* [Join the Ubuntu HPC community on Matrix](https://matrix.to/#/#hpc:ubuntu.com)
 * [Get the latest news on Discourse](https://discourse.ubuntu.com/c/hpc/151)
-* [Ask and answer questions on GitHub](https://github.com/orgs/charmed-hpc/discussions/categories/q-a)
+* [See our Contributing guide](https://github.com/charmed-hpc/.github/blob/main/CONTRIBUTING.md)
```

Comment:
I'd rather not link directly to a "bespoke" file on GitHub. It feels unprofessional especially if you're not someone who's _immersed_ in hacker culture. I'd be concerned that someone who's evaluating Charmed HPC from a purchasing standpoint might see this as a quick hack.

I think this URL is captured by my suggestion below. If someone's interested in contributing directly to Charmed HPC, they'll go to the GitHub organization, see the _excellent_ profile README, and get the link to the global CONTRIBUTING.md document from there. I'd only include a direct link to a contributing guide here if it was hosted directly on our RTD site as a nicely rendered page rather than a single file on GitHub.